python/spacewalk/spacewalk-backend.spec

```diff
@@ -69,6 +69,7 @@ Requires(pre):  %{apache_pkg}
 Requires:       %{apache_pkg}
 Requires:       python3-pycurl
 Requires:       python3-libmodulemd
+Requires:       python3-looseversion
```

Comment:
@agraul are we  good here just requiring `python3-looseversion` or also here we need  the same dance with the use_python macro like we also did in spacewalk-sls for requiring  python311-salt?

Reply:
`python3-looseversion` is enough, it's used in reposync (running with Python 3.6). That makes me think we should move this to `%package tools` if we want to be correct. But that's mostly cosmetic as both packages will be installed into the container image either way